tests/Integration/DataManagementSystem/Test_DataManager.py

```diff
@@ -0,0 +1,207 @@
+from __future__ import print_function
```

Comment:
Please add a few lines of docstring, also writing what's needed for this test to be run

Reply:
Added docstring and requirements